src/Sarif.Viewer.VisualStudio.UnitTests/ErrorList/SarifErrorListItemTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif;
+using Xunit;
+
+namespace Microsoft.Sarif.Viewer.VisualStudio.UnitTests
+{
+    public class SarifErrorListItemTests
+    {
+        [Fact]
+        public void SarifErrorListItem_LineMarker_Region_StartLine()
+        {
+            new SarifViewerPackage();
```

Comment:
>SarifViewerPackage [](start = 16, length = 18)

Is this something you saw in other unit tests? What does it do? It must have a side effect, since you're discarding the resulting object. Does the test work without this line?

Reply:
The constructor sets a static variable in the class, which is used in the method I'm testing. Without it, it throws a null reference exception.